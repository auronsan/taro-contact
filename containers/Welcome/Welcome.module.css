.title {
  color: light-dark(var(--color-black), var(--color-white));
  font-size: rem(100px);
  font-weight: 900;
  letter-spacing: rem(-2px);

  @media (max-width: $breakpoint-md) {
    font-size: rem(50px);
  }
}
