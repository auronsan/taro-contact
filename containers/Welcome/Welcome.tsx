import { Anchor } from '@/components/Anchor';
import { Text } from '@/components/Text';
import { Title } from '@/components/Title';
import classes from './Welcome.module.css';

export function Welcome() {
  return (
    <>
      <Title className={classes.title}>
        Welcome to{' '}
        <Text component="span" gradient={{ from: '#93C04B', to: '#08c654' }} size="inherit">
          Taro Contact
        </Text>
      </Title>
      <Text ta="center" size="lg" component="p">
        <Anchor href="/contacts" size="lg">
          Go to Contacts
        </Anchor>
      </Text>
    </>
  );
}
