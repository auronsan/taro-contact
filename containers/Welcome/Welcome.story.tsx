import { Welcome } from './Welcome';

export default {
  title: 'containers/Welcome',
  component: Welcome,
};

export const Usage = () => <Welcome />;
