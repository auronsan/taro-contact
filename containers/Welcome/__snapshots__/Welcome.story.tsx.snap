// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/Welcome Usage smoke-test 1`] = `
<main class="roboto-normal">
  <div style="display: flex; justify-content: center; align-items: center; height: 100vh; flex-direction: column;">
    <h2 class="title_title__RhqmI Welcome_title__IQrvc">
      Welcome to
      <span class="text_text__ynNTn text_text-gradient__MXfWo text_fs-inherit__f2K_J"
            style="background-image: linear-gradient(0deg, rgb(147, 192, 75) 0%, rgb(8, 198, 84) 100%);"
      >
        Taro Contact
      </span>
    </h2>
    <p class="text_text__ynNTn text_ta-center__Rbgq3 text_fs-lg__IAQ1L">
      <a href="/contacts"
         class="anchor_root__e1GyS anchor_fs-lg__D_wUE"
      >
        Go to Contacts
      </a>
    </p>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;
